pkgs/top-level/python-packages.nix

```diff
@@ -8035,12 +8037,16 @@ in {
     };
   };
 
+  flask_common = callPackage ../development/python-modules/flask-common { };
+
   flask-compress = callPackage ../development/python-modules/flask-compress { };
 
   flask-cors = callPackage ../development/python-modules/flask-cors { };
 
   flask_elastic = callPackage ../development/python-modules/flask-elastic { };
 
+  flask_limiter = callPackage ../development/python-modules/flask-limiter { };
```

Comment:
Since the canonical PyPI names of the new packages are Flask-Common and Flask-Limiter, and the manual says not to convert hyphens to underscores (https://nixos.org/nixpkgs/manual/#sec-package-naming), could you use hyphenated names?

Reply:
Actually, in `python-packages.nix` we do not follow those guidelines, but instead stick to the canonical PyPI names, or the normalized names (https://www.python.org/dev/peps/pep-0503/#normalized-names)

Reply:
At least, that's what I am pushing for.